### ISSUE TITLE
Use multi-threading within Encoder.py to speed up compression time

### ISSUE THREAD

**Issue by tim-depthkit** [XRFoundation/UniversalVolumetric#198]
It would be nice to have the option to run the encoder with multiple threads in order to run the draco_encoder and basisu subprocesses in parallel. This could be accomplished with the python threading module.

**Comment by the-reality-engineer** [comment]
Hi Tim! Thanks for opening the issue! Yes, we will be working towards full-featured encoding software that efficiently encodes to multiple targets, including both Geometry and Texture LODs generated from source assets.
